### ISSUE TITLE
Add functionality to maintain (custom) functions

### ISSUE THREAD

**Issue by Zefau** [Zefau/ioBroker.jarvis#188]

**Comment by duczz** [comment]
manuell editing works currently too - as workaround?

**Comment by Zefau** [comment]
What do you mean? There is currently no workaround for editing manually.

**Comment by duczz** [comment]
sure in Developer mode, then you have a new category in the layout overview

**Comment by Zefau** [comment]
yeah, that is true, you may specify a new function via the json editor

**Comment by duczz** [comment]
makes no touble in future changes?
